operator/standalone/delete_it_test.go

```diff
@@ -0,0 +1,288 @@
+////go:build integration
+
+package standalone
+
+import (
+	"context"
+	"fmt"
+	"github.com/vshn/appcat-service-postgresql/apis/postgresql/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+
+	pipeline "github.com/ccremer/go-command-pipeline"
+	helmv1beta1 "github.com/crossplane-contrib/provider-helm/apis/release/v1beta1"
+	"github.com/stretchr/testify/suite"
+	"github.com/vshn/appcat-service-postgresql/operator/operatortest"
+)
+
+type DeleteStandalonePipelineSuite struct {
+	operatortest.Suite
+}
+
+func TestDeleteStandalonePipeline(t *testing.T) {
+	suite.Run(t, new(DeleteStandalonePipelineSuite))
+}
+
+func (ts *DeleteStandalonePipelineSuite) BeforeTest(suiteName, testName string) {
+	ts.Context = pipeline.MutableContext(context.Background())
+	setClientInContext(ts.Context, ts.Client)
+	ts.RegisterScheme(helmv1beta1.SchemeBuilder.AddToScheme)
+}
+
+func (ts *DeleteStandalonePipelineSuite) Test_DeleteHelmRelease() {
+	tests := map[string]struct {
+		prepare                    func(releaseNameString string)
+		givenReleaseName           string
+		expectedHelmReleaseDeleted bool
+		expectedError              error
```

Comment:
I would recommend to check the error message as well, not just whether it's nil or not.
```suggestion
		expectedError              string
```

Reply:
Updated all tests to check on the message string instead of the error itself